### PR TITLE
Add component CommentsList

### PR DESCRIPTION
**Pull request by andreszuluaga-wolox**
## Summary

Add component CommentsList.

#### Galaxy S (or similar)
WhatsApp Image 2020-12-30 at 3 45 11 PM

#### Pixel 2/3 (or similar)
Screenshot_1609360736

#### Pixel 3XL (or similar)
Screenshot_1609360842

#### iPhone SE (or similar)
simulator_screenshot_0FA1C43D-0B79-4C91-9899-D82DF93CA283

#### iPhone 8 (or similar)
simulator_screenshot_D342D054-71F9-4909-9D90-9B81BA0A4BB7

#### iPhone 11 (or similar)
simulator_screenshot_FB1485F8-8DC2-41C0-9BB5-6653D492ACEF

## Trello Card

https://trello.com/c/rhZtfF80/16-n-73-tabla-de-comentarios

### CODE REVIEW

File: wbooks/src/app/components/CommentsList/index.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import styles from './styles';
+
+function CommentList() {
+  const [limit, setLimit] = useState(2);
+  const renderItem: ListRenderItem<Comments> = ({ item }: { item: Comments }) => {
+    return <CommentsBook {...item} />;
+  };
+  const keyExtractor = (item: Comments) => `${item.id}`;
+  const viewAll = () => {
+    setLimit(5);
```
Comment (guidoprinc) [inline wbooks/src/app/components/CommentsList/index.tsx:16]: this should show all (the maximum amount), not just 5

File: wbooks/src/app/components/CommentsBook/index.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <View style={styles.imageContainer}>
+          {url === null ? (
+            <View style={styles.image} />
+          ) : (
+            <Image source={{ uri: url }} style={styles.imageContainer} />
+          )}
+        </View>
+        <View style={styles.textContainer}>
+          <Text style={styles.title}>{author}</Text>
+          <Text style={styles.author}>{text}</Text>
```
Comment (guidoprinc) [inline wbooks/src/app/components/CommentsBook/index.tsx:20]: use appropriate names for this styles

Comment (guidoprinc) [reply wbooks/src/app/components/CommentsBook/index.tsx:20]: check the image, they are not the same as the BookItem

File: wbooks/src/app/components/CommentsBook/styles.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    marginHorizontal: 20,
+    marginVertical: 10,
+    flexDirection: 'row'
+  },
+  imageContainer: {
+    alignSelf: 'center'
+  },
+  image: {
+    width: 50,
+    height: 50,
```
Comment (guidoprinc) [inline wbooks/src/app/components/CommentsBook/styles.ts:15]: extract this common constant inside this file

Comment (guidoprinc) [reply wbooks/src/app/components/CommentsBook/styles.ts:15]: use it for `borderRadius` too

**Review by guidoprinc — CHANGES_REQUESTED** [review]
I think that the comments list should be below the detail of the book, not in another screen. Check that

File: wbooks/src/app/components/CommentsList/index.tsx (hunk 1 above)
Comment (andreszuluaga-wolox) [reply wbooks/src/app/components/CommentsList/index.tsx:16]: Ok. The card say, last 5. But not import, I fix it

File: wbooks/src/app/index.tsx [hunk 4]
```diff
@@ -26,7 +27,8 @@ const Stack = createStackNavigator();
 const App = () => {
   return (
     <NavigationContainer>
-      <Stack.Navigator initialRouteName="BookList">
+      <Stack.Navigator initialRouteName="CommentsList">
+        <Stack.Screen name="CommentsList" component={CommentsList} />
```
Comment (guidoprinc) [inline wbooks/src/app/index.tsx:31]: this should be a component inside bookdetail

**Review by guidoprinc — CHANGES_REQUESTED** [review]

File: wbooks/src/app/index.tsx (hunk 4 above)
Comment (guidoprinc) [reply wbooks/src/app/index.tsx:31]: remove this line

**Review by guidoprinc — CHANGES_REQUESTED** [review]

**Review by guidoprinc — APPROVED** [review]

**Pull request merged**